docker-cloud/cloud-swarm/using-swarm-mode.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Work with swarms in Docker Cloud
+keywords: swarm mode, swarms, orchestration Cloud, fleet management
+title: Using swarm mode with Docker Cloud
+---
+
+Docker Cloud now allows you to connect to clusters of Docker Engines running in v1.13 [swarm mode](/engine/swarm/).
+
+## Swarm mode
+
+The current release of Docker Cloud automatically opts in new users to Beta
+Swarm mode.
+
+### New to Docker Cloud?
+
+If you just signed up as a new Docker Cloud user, Swarm mode is baked into the
+standard web interface. This is leading edge functionality for Docker Cloud and
```

Comment:
Remove: "This is leading edge functionality for Docker Cloud and we plan to build out swarm management features using this model in future releases."